lumicks/pylake/population/mixture.py

```diff
@@ -78,27 +81,18 @@ def __init__(self, data, n_states, init_method, n_init, tol, max_iter):
         )
 
     @classmethod
+    @deprecated(
+        reason=(
+            "This method has been deprecated and will be removed in a future version. You can now "
+            "use `Slice` instances to construct this class directly."
+        ),
+        action="always",
```

Comment:
Please update the docs to actually reflect this. The method `from_channel` is still used in `population.rst`.

Reply:
oops, I noticed this but forgot to push the fixup commit